lib/matplotlib/axes/_base.py

```diff
@@ -2032,14 +2032,14 @@ def add_container(self, container):
         container._remove_method = self.containers.remove
         return container
 
-    def _on_units_changed(self, scalex=False, scaley=False):
+    def _on_units_changed(self, **kwargs):
         """
         Callback for processing changes to axis units.
 
-        Currently forces updates of data limits and view limits.
+        Currently requests updates of data limits and view limits.
         """
         self.relim()
-        self._request_autoscale_view(scalex=scalex, scaley=scaley)
+        self._request_autoscale_view(**kwargs)
```

Comment:
The default values for `_request_autoscale_view` are `True` while the defaults for this method are `False`. Unless all the callers of `_on_units_changed` are modified, this will not trigger the same behaviour.

Reply:
Ugh, good catch.  There are possible fixes but they're a bit ugly, so I just reverted this for now.